docs/source/benchmarking.rst

```diff
@@ -127,11 +77,92 @@ Example for the 78Mbp simulated community
     gdown https://drive.google.com/uc?id=${file_id} -O metagenome.fna.gz
 
 .. note::
+    
+    1. To follow along with the tutorial below you'll need to download ``reference_assignments.tsv.gz``, ``binning.tsv.gz``, and ``taxonomy.tsv.gz`` files for the dataset you want to run benchmarking on.
 
-    Unfortunately, at the moment ``gdown`` doesn't support downloading entire directories from Google drive.
+    2. Unfortunately, at the moment ``gdown`` doesn't support downloading entire directories from Google drive.
     There is an open `Pull request <https://github.com/wkentaro/gdown/pull/90#issue-569060398>`_ on the ``gdown`` repository
     addressing this specific issue which we are keeping a close eye on and will update this documentation when it is merged.
 
+Benchmark clustering
+^^^^^^^^^^^^^^^^^^^^
+
+Here we are benchmarking the clustering step. All the simulated communities are being benchmarked. You don't have to run all of them, this is just for tutorial purposes.
```

Comment:
I think this simply needs to show usage rather than be a full-fledged tutorial. If that is something we would like to do, we should do it in the [KwanLab/metaBenchmarks](https://github.com/KwanLab/metaBenchmarks "https://github.com/KwanLab/metaBenchmarks") repo